server/user/datasheet.js

```diff
@@ -7,7 +7,8 @@ const Worksheet = require('exceljs/dist/es5/doc/worksheet');
 
 const isCsv = file => file.mimetype === 'text/csv';
 const isString = arg => typeof arg === 'string';
-const parseCell = cell => cell.isHyperlink ? cell.value.text : cell.value;
+const parseCell = cell => cell.isHyperlink ? parseCellText(cell.value.text) : cell.value;
+const parseCellText = cellText => cellText.richText ? cellText.richText[0].text : cellText;
```

Comment:
Why to use only first chunk of rich text? Is joining text property of all rich text items an option?

Reply:
Probably.

Reply:
@mcapeta this has been addressed.